x-pack/filebeat/docs/inputs/input-http-endpoint.asciidoc

```diff
@@ -273,6 +274,11 @@ This option copies the raw unmodified body of the incoming request to the event.
 
 This option defines the provider of the webhook that uses CRC (Challenge-Response Check) for validating the endpoint. The HTTP endpoint input is responsible for ensuring the authenticity of incoming webhook requests by generating and verifying a unique token. By specifying the `crc.provider`, you ensure that the system correctly handles the specific CRC validation process required by the chosen provider.
 
+[float]
+==== `crc.secret`
+
+The secret token provided by the webhook owner for the CRC validation.
```

Comment:
This should note that it is required.

Reply:
Done https://github.com/elastic/beats/pull/35692. Thanks!